### PR TITLE
Update cssom.idl and work around removal of CSSOMString

### PR DESCRIPTION
**Pull request by foolip**
Fixes https://github.com/web-platform-tests/wpt/issues/12515.

### CODE REVIEW

**Comment by foolip** [comment]
https://github.com/web-platform-tests/wpt/pull/15071 needs to be merged first and the tests for this then run again. Should have done a single PR :)

File: resources/idlharness.js [hunk 1]
```diff
@@ -3152,11 +3152,25 @@ function idl_test(srcs, deps, idl_setup_func) {
...
+            // CSSOMString is an implementation-defined typedef of either
+            // DOMString or USVString in CSSOM:
+            // https://drafts.csswg.org/cssom/#cssomstring-type
+            // Because of this, neither typedef is included in cssom.idl.
+            // We need some definition, and use DOMString because USVString
+            // has additional requirements in type conversion and could result
+            // in spurious failures for implementations that use DOMString.
+            // Note: No newline is added to preserve line numbers.
+            return 'typedef DOMString CSSOMString;' + idl;
+        }
```
Comment (Ms2ger) [inline resources/idlharness.js]: This is an odd place to put it; can we just always add it in the `IdlArray` constructor?

Comment (lukebjerring) [reply resources/idlharness.js]: I don't think special-casing it in the testing framework is the right approach. The fact that it's defined somewhere in the bikeshed source (differently than an IDL snippet) should be handled by the scraper, or importer, but not by the idlharness or test layer.

**Comment by foolip** [comment]
New deal: https://github.com/tidoust/reffy-reports/pull/29

**Pull request closed without merging**